src/org/openstreetmap/josm/plugins/pt_assistant/actions/CreatePlatformNodeAction.java

```diff
@@ -390,17 +390,4 @@ protected void buttonAction(int buttonIndex, ActionEvent evt) {
 
 		}
 	}
-
-	private void initialiseKeys () {
```

Comment:
Since there are default values this is not needed. The properties moved to the class `PTProperties`, where in the long run all properties should be located.